charts/opentelemetry-collector/values.yaml

```diff
@@ -442,6 +442,25 @@ readinessProbe:
     port: 13133
     path: /
 
+# startup probe configuration
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+##
+startupProbe:
+  {}
```

Comment:
No idea why the formatter wants to put this on a new line. Only happens when the commented text is below..